corehq/apps/tour/utils.py

```diff
@@ -0,0 +1,9 @@
+from corehq.apps.tour.models import GuidedTours
+
+
+def activate_tour_for_user(tour_slug, user):
+    GuidedTours.enable_tour_for_user(tour_slug, user.username)
+
+
+def tour_is_enabled_for_user(tour_slug, user):
+    return GuidedTours.is_tour_enabled_for_user(tour_slug, user)
```

Comment:
Non-blocking: what do you think of moving these into the `tours` module rather than here in this `utils` module?

Seems like they are closer to core app functionality than utilities. Utility modules tend to become a conglomeration of unrelated junk that didn't fit anywhere, so I think it's better to avoid them when there is another place where it would make sense to put something.